dspace-api/src/main/java/org/dspace/content/packager/AbstractMETSDisseminator.java

```diff
@@ -818,11 +818,7 @@ protected Mets makeManifest(Context context, DSpaceObject dso,
         Mets mets = new Mets();
         
         String identifier = "DB-ID-" + dso.getID();
-        if(dso.getHandle()!=null)
-        {
-            identifier = dso.getHandle().replace('/', '-');
-        }
-        
+
```

Comment:
It's unclear to me why this `if` statement is being removed.  Could you explain this further?

Reply:
I needed a place to store the object UUID. Because the handle is already stored (and - more specifically - read) from the `OBJID` field, I removed this `if` block in order to be sure the UUID would be present in the `ID` field.